test/types/index.ts

```diff
@@ -259,7 +260,12 @@ server.setErrorHandler((err, request, reply) => {
 
 server.listen(3000, err => {
   if (err) throw err
-  server.log.info(`server listening on ${server.server.address().port}`)
+  const address = server.server.address()
+  if (typeof address === 'object') {
+    server.log.info(`server listening on ${(<AddressInfo>server.server.address()).port}`)
```

Comment:
Why you have to cast it? The typings should point to the correct type.

Reply:
Sorry. I should use the `address` variable directly. Thanks for reviewing it

Reply:
Fixed